src/main/resources/application.conf

```diff
@@ -0,0 +1,15 @@
+akka {
+  event-handlers = ["akka.event.slf4j.Slf4jEventHandler"]
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  loglevel = "DEBUG"
+  actor {
+    debug {
+      //autoreceive = on
+      //lifecycle = on
+      receive = on
+      event-stream = on
+      fsm = on
+      //unhandled = on
```

Comment:
should the commented-out parameters be removed or are these things we're likely to want to toggle locally?